test/src/test/java/org/corfudb/infrastructure/LayoutBuilderTest.java

```diff
@@ -245,4 +246,55 @@ public void checkAdditionOfNodes() throws Exception {
 
         assertThat(layoutBuilder.build()).isEqualTo(expectedLayout);
     }
+
+    /**
+     * Reassign a failover sequencer server from the set of responsive nodes.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void reassignResponsiveSequencer() throws Exception {
+
+        Layout originalLayout = new TestLayoutBuilder()
+                .setEpoch(1L)
+                .addLayoutServer(SERVERS.PORT_0)
```

Comment:
Although it is a test, it might be nice to have  one or two helper methods to avoid the repetition of layout building. Your decision as it is worth it or not.

Reply:
I assessed the suggestion of adding helper method. It doesn't carry its weight at current state. This is due to different variations that it has to support for creating different Layouts used in different tests.